include/constants.h

```diff
@@ -6,20 +6,90 @@
 #define INCLUDE_CONSTANTS_H_
 
 #include <string>
+#include <regex>
+#include <tgbot/tgbot.h>
 
 namespace consts {
 // TODO(diduk001): проверять регексом в compile-time
 constexpr char TOKEN[] = "...";
 constexpr char DB_NAME[] = "../chinook.db";
 }  // namespace consts
-
 namespace messages {
-constexpr char HI[] = "Hi!";
+constexpr char HI[] = "Здарова шершень";
+// making
+constexpr char discipline_making_new[] = "Вы создаете новую дисциплину";
+constexpr char discipline_making_canceled[] = "Создание дисциплины отменено";
+constexpr char discipline_is_saved[] = "Дисциплина сохранена в базе данных";
+//description
+constexpr char discipline_have_no_description[] = "В дисциплине не будет описания";
+constexpr char discipline_is_correct_description[] = "Информация записана правильно?";
+// enter
+constexpr char discipline_enter_name[] = "Введите название дисциплины";
+constexpr char discipline_enter_full_name[] = "Введите ФИ(О) преподавателя";
+constexpr char discipline_enter_email[] = "Введите почту преподавателя";
+constexpr char discipline_enter_description[] = "Введите описания предмета";
+// saved
+constexpr char discipline_professor_name_saved[] = "Имя преподавателя сохранено";
+constexpr char discipline_name_saved[] = "Название дисциплины сохранено";
+constexpr char discipline_email_saved[] = "Электронная почта сохранена";
+constexpr char discipline_saved[] = "Проверьте правильность данных:";
+// wrong
+constexpr char discipline_wrong_name_length[] = "Неправильная длина дисциплины\nДлина должна быть более: ";
+constexpr char discipline_wrong_full_name[] = "Неправильный формат ввода ФИ(О)";
+constexpr char discipline_wrong_email[] = "Проверьте правильность ввода почты, впишите ее еще раз";
+constexpr char discipline_wrong_describe[] = "Максимальная длина для описания {}";
+
+constexpr char discipline_what_want_to_do[] = "Что вы хотите сделать?";
 }
 
 namespace db_queries {
 constexpr char basic_where_from_customers[] =
     "SELECT FirstName FROM customers WHERE Country=?";
 }
 
+namespace regular_expressions {
+const std::regex pattern(R"(\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b)");
```

Comment:
https://clehaxze.tw/gemlog/2023/01-28-validate-email-address-using-regex-in-cpp.gmi